docs/analytics/screen-tracking.md

```diff
@@ -38,15 +40,18 @@ documentation on the React Navigation website.
 
 The [`wix/react-native-navigation`](https://github.com/wix/react-native-navigation) provides 100% native platform navigation
 for React Native apps. To manually track screens, you need to setup a `componentDidAppear` event listener and manually call the
-[`setCurrentScreen`](/reference/analytics#setCurrentScreen) method the Analytics library provides:
+[`logScreenView`](/reference/analytics#logScreenView) method the Analytics library provides:
 
 ```js
 import analytics from '@react-native-firebase/analytics';
 import { Navigation } from 'react-native-navigation';
 
 Navigation.events().registerComponentDidAppearListener(({ componentName, componentType }) => {
   if (componentType === 'Component') {
-    analytics().setCurrentScreen(componentName, componentName);
+    analytics().logScreenView({
```

Comment:
Should probably make both the example functions async and await the call of logScreenView 😅